android/src/main/java/guichaguri/trackplayer/player/players/AndroidPlayback.java

```diff
@@ -90,6 +91,13 @@ public void load(Track track, Promise callback) {
             if(player.isPlaying()) player.stop();
             player.reset();
             player.setDataSource(context, url);
+
+            if (earpiece == true) {
+                player.setAudioStreamType(AudioManager.STREAM_VOICE_CALL);
+            } else {
+                player.setAudioStreamType(AudioManager.STREAM_MUSIC);
+            }
```

Comment:
Does it have set the stream type before it loads the track?

Reply:
Not sure what do you think.

Reply:
Well, if you ask about some default value. There is a line 41: `private boolean earpiece = false`. So STREAM_MUSIC will be set up by default .. Like it was before ..